code/io/joy-sdl.cpp

```diff
@@ -490,7 +488,31 @@ namespace joystick
 
 			if (_hat[i].Value != HAT_CENTERED)
 			{
-				_hat[i].DownTimestamp = timer_get_milliseconds();
+				// Split corners
```

Comment:
Same as above. The joystick API is meant to use the SDL API to its full potential.